pkg/agent/cniserver/sriov.go

```diff
@@ -0,0 +1,130 @@
+package cniserver
+
+import (
+	"fmt"
+	"context"
+	"net"
+	"os"
+	"path"
+	"time"
+	"google.golang.org/grpc"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"antrea.io/antrea/pkg/agent/util"
+	podresourcesv1alpha1 "k8s.io/kubelet/pkg/apis/podresources/v1alpha1"
+	sriovcniutils "github.com/k8snetworkplumbingwg/sriov-cni/pkg/utils"
+	"k8s.io/klog/v2"
+)
+
+const (
+	kubeletPodResourcesPath = "/var/lib/kubelet/pod-resources"
+	kubeletSocket           = "kubelet.sock"
+	connectionTimeout       = 10 * time.Second
+	maxMsgSize              = 1024 * 1024 * 16 // 16 Mb
+)
+
+type KubeletPodResources struct {
+	resources []*podresourcesv1alpha1.PodResources
```

Comment:
I notice we use version v1alpha1 of the API. Do you know what's the minimum version of K8s required in order to use v1 instead? Maybe worth adding a comment.

Reply:
Kubelet v1 API requires minimum v1.20.0; will add a comment to that effect. Thanks.